db/dcrpg/upgrades.go

```diff
@@ -1271,6 +1420,19 @@ func deleteRootsColumns(db *sql.DB) (bool, error) {
 	return deleteColumnsIfFound(db, table, cols)
 }
 
+func deleteVoteMilestoneColumns(db *sql.DB) (bool, error) {
+	table := "agendas"
```

Comment:
There's no `table` variable in `addVotesBlockTimeColumn`, and for consistency I suggest eliminating the variable and passing a string literal.